fire/completion.py

```diff
@@ -326,7 +326,7 @@ def MemberVisible(component, name, member, class_attrs=None, verbose=False):
       # Methods and properties should only be accessible on instantiated
       # objects, not on uninstantiated classes.
       if class_attr.kind in ('method', 'property'):
-        return False
+        return True
```

Comment:
Can you provide an example of why you want to make this change?
The comment at 326-327 suggests that False was intended.

If we are changing this to True, we'll want to change that comment as well.

Reply:
Sorry, I see you already have done so in #225. My bad.